.github/workflows/publish-source.yml

```diff
@@ -2,6 +2,13 @@
 name: Publish source
 on:
   create
+
+# Detect if this action is already running, and cancel it.
+# This most likely happened because a second push has been made to a branch.
+concurrency:
+  group: ${{ github.repository_id }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
I think it as a bad idea to use it on the translation system. Maybe I am wrong, but I think it could have unforeseen consequences . So I would rather leave it without the concurrency functionality.